data/competencies.yml

```diff
@@ -1,16 +1,292 @@
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: da526878-694a-4a73-b678-f3e46301e1b6
+  summary: Presents their own work clearly to a product owner or tech lead
+  examples:
+    - Talks about their progress in standup without going into deep technical detail
+    - Explains their approach to a technical problem to their tech lead
+    - Gives a live demo of a feature that they worked on to their team
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+- id: b8bd799d-6712-4882-b1a7-4f66166c3315
+  summary: Effectively communicates when they are unable to attend expected events
+  examples:
+    - Informs meeting organisers of meetings they cannot attend
+    - Puts out-of-office messages on emails when going on holiday
+    - Puts appropriate emoji on Slack when sick or on holiday
+    - If they notice a meeting they've been invited to clashes with another meeting, they inform the organiser of the meeting they have to miss
+    - If a meeting is likely to overrun into another meeting they communicate both to the first meeting's host that they have other meeting and need to drop off, and explain to the second meeting's organiser they may be late
+    - If one recurring meeting regularly overruns into another they additionally escalate to their line manager to help resolve the situation
+  supportingUrls: []
+  description: null
+  level: junior
+  area: communication
+  domain: null
+
+# Junior Engineer, Delivery
+
+- id: 05402e55-07c6-4755-9e93-240dfc656255
+  summary: Recognises and communicates conflicting priorities and delivery expectations
+  examples:
+    - Recognises when meetings clash with other responsibilities, and communicates to affected parties with the aim of resolving the conflict
+    - Communicates the difficulties they encounter with delivering a ticket rather than struggling on their own
+  supportingUrls: []
+  description: null
+  level: junior
+  area: delivery
+  domain: null
+
+- id: ed6d6a37-08f3-403c-a350-fef1652d7903
+  summary: Attends their team's meetings, and shares their views and opinions
+  examples: []
+  supportingUrls: []
+  description: null
+  level: junior
+  area: delivery
+  domain: null
+
+- id: 1f5273b2-d9e1-459b-b0aa-6e9379b49bbc
+  summary: Regularly communicates the status of their work
+  examples: []
+  supportingUrls: []
+  description: null
+  level: junior
+  area: delivery
+  domain: null
+
+- id: 12125116-15de-47c6-81f5-449c3b07a304
+  summary: Participates in the delivery process
+  examples:
+    - Moves tickets to done column when they are complete
+    - Goes to stand-ups and communicates progress
+  supportingUrls: []
+  description: null
+  level: junior
+  area: delivery
+  domain: null
+
+- id: 70f624cf-d634-45f5-ac81-2b73768cd792
+  summary: Knows who their project's stakeholders are and references them where appropriate
```

Comment:
At junior level is it better to emphasise that they understand the purpose/objective/goals of the team/project?

Reply:
That [pull request is now here](https://github.com/Financial-Times/engineering-progression/pull/360).